src/Monocle/Logging.hs

```diff
@@ -62,17 +63,16 @@ data LogEvent
   | AddingTaskData LText Int
   | UpdatingEntity LText Entity UTCTime
   | Searching QueryRequest_QueryType LText Q.Query
-  | SystemReady Int Int Text
   | AuthSystemReady Text
-  | ReloadConfig FilePath
-  | RefreshIndex Config.Index
   | OIDCCallbackCall (Maybe Text) (Maybe Text) (Maybe Text)
   | OIDCProviderTokenRequested Text
-  | JWTCreated Text Text
-  | JWTCreateFailed Text Text
+  | LogStructured Text [Aeson.Pair]
 
 instance From LogEvent Text where
   from = \case
+    LogStructured msg attrs
```

Comment:
I believe that info like "System ready" which is passed via msg here and the LogAuthor must be mandatory fields of the LogStructured type. It seems that the intend is to make logs processable via a program then each useful info must be part of the JSON struct.

Reply:
Well the current SystemReady event doesn't have a LogAuthor. I think what we want instead is the source location of the message, e.g. `Monocle.Api`, `Macroscope.Worker` or `Lentille.GitHub`, and we will get that automically by using [HasCallStack](https://hackage.haskell.org/package/base-4.13.0.0/docs/GHC-Stack.html#t:HasCallStack).

For crawler information, these should be added as attrs, e.g. `["crawler" .= crawlerName, "entity" .= entity]`